src/crudOps.js

```diff
@@ -0,0 +1,76 @@
+// JS to handle tsk list array
+import Storage from './storage.js';
+import menu from './assets/images/3-dots.png';
+
+const toDoList = document.querySelector('.to-do-list-ul');
+
+export const addToDo = (value) => {
+  let arr;
+  if (Storage.getLocalStorage() == null) {
+    arr = [];
+  } else {
+    arr = Storage.getLocalStorage();
+  }
```

Comment:
- Please @TemesgenA I tried adding todos to the todo list but it does not seem to create the list nor the local storage,  Please try to link the files correctly and request a review.